src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -14,15 +15,23 @@ internal partial class LocalRewriter
     {
         public override BoundNode VisitConversion(BoundConversion node)
         {
-            var rewrittenType = VisitType(node.Type);
             if (node.ConversionKind == ConversionKind.InterpolatedString)
             {
                 return RewriteInterpolatedStringConversion(node);
             }
 
+            var rewrittenType = VisitType(node.Type);
+            var rewrittenOperand = node.Operand;
+
```

Comment:
This assignment is super confusing. You are assigning the un-rewritten operand to a variable named rewrittenOperand.